CAIPs/caip-24.md

```diff
@@ -0,0 +1,70 @@
+---
+caip: 24
+title: Asset Reference for the cosmos Asset Namespace
+author: Theobold Frazier
+discussions-to: https://github.com/shapeshift/CAIPs/issues/1
+status: Draft
+type: Standard
+created: 2020-02-14
+updated: 2020-02-14
+requires: 19
+---
+
+## Simple Summary
+
+This document is about the details of the Cosmos network asset namespace and reference for CAIP-19.
+
+## Abstract
+
+In CAIP-19 a general asset identification scheme is defined. This is the
+implementation of CAIP-19 for Cosmos chain native assets and ibc assets.
+
+## Motivation
+
+See CAIP-19.
+
+## Specification
+
+### Cosmos chain Native Asset Namespace
+
+An asset namespace called "native" that references cosmos network blockchain native
+assets, allowing for unique identification on cosmos chains with multiple native assets.
+
+### Cosmos chain ibc asset Asset Namespace
```

Comment:
Is `asset Asset` intentional?

Reply:
It was, but now I don't like it. Changed it up.